plugin/storage/kafka/options.go

```diff
@@ -69,12 +75,30 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultEncoding,
 		fmt.Sprintf(`(experimental) Encoding of spans ("%s" or "%s") sent to kafka.`, EncodingJSON, EncodingProto),
 	)
+	flagSet.Int(
+		configPrefix+suffixRequiredAcks,
+		defaultRequiredAcks,
+		"(experimental) Required kafka broker acknowledgement. default = 1, no response = 0, wait for local = 1, wait for all = -1",
+	)
+	flagSet.Int(
+		configPrefix+suffixCompression,
+		defaultComression,
+		"(experimental) Type of compression to use on messages. default = 0, none = 0, gzip = 1, snappy = 2, lz4 = 3, zstd = 4",
+	)
+	flagSet.Int(
+		configPrefix+suffixCompressionLevel,
+		defaultCompressionLevel,
+		"(experimental) Level of compression to use on messages. default= -1000, gzip = 1-9 (default = 6), snappy = none, lz4 = 1-17 (default = 9), zstd = -131072 - 22 (default = 3)",
```

Comment:
Sorry didn't pick up on this before, but there isn't a single default for this property - there is a default per compression type. So the default will need to be determines based on the value of the `compression` flag. And then have a test that checks the default for each of the valid `compression` values.

Reply:
I have followed below kafka documentation for compression level.
https://cwiki.apache.org/confluence/display/KAFKA/KIP-390%3A+Allow+fine-grained+configuration+for+compression
Default values are based on current compression libraries followed in Kafka.